pkg/agentapiclient/tunnel.go

```diff
@@ -9,52 +9,59 @@ import (
 	"time"
 )
 
-const tunnelApiUrl = "http://localhost:4040/api/tunnels"
+const tunnelAPIURL = "http://localhost:4040/api/tunnels"
```

Comment:
Api -> API and Url -> URL are the idiomatic spellings. Go generally prefers acronyms to be capitalized, which is weird and different from every other language, but that's the idiom.

See https://github.com/golang/go/wiki/CodeReviewComments#initialisms